controllers/model/grafana_resources.go

```diff
@@ -118,6 +118,15 @@ func GetGrafanaResourceRequirement(cr *v1.Observability) *v14.ResourceRequiremen
 	return &v14.ResourceRequirements{}
 }
 
+func GetGrafanaVersion(indexes []v1.RepositoryIndex) string {
+	config := indexes[0].Config
+	if config != nil && config.Grafana.GrafanaVersion != "" {
+		return config.Grafana.GrafanaVersion
+	}else{
+		return ""
```

Comment:
If no version is specified in the config, does it revert to a specific version/latest?

Reply:
So if no version is specified ie its left blank it will not set it to different image and uses the one which is defined within the Grafana Operator which at the moment is version 7.3.10 I believe.

Reply:
I see. Just one more question: do we need the else here? Or could we do something like:
```
if config != nil && config.Grafana.GrafanaVersion != "" {
		return config.Grafana.GrafanaVersion
	}
          return ""
```